spec/grape/validations/validators/length_spec.rb

```diff
@@ -0,0 +1,221 @@
+# frozen_string_literal: true
+
+describe Grape::Validations::Validators::LengthValidator do
+  let_it_be(:app) do
+    Class.new(Grape::API) do
+      params do
+        requires :list, length: { min: 2, max: 3 }
+      end
+      post 'with_min_max' do
+      end
+
+      params do
+        requires :list, type: [Integer], length: { min: 2 }
+      end
+      post 'with_min_only' do
+      end
+
+      params do
+        requires :list, type: [Integer], length: { max: 3 }
+      end
+      post 'with_max_only' do
+      end
+
+      params do
+        requires :list, type: Integer, length: { max: 3 }
+      end
+      post 'type_is_not_array' do
+      end
```

Comment:
This is confusing. What is the length of an integer?

Reply:
yup now if the type is not string or array it will raise an `ArgumentError`.